ckan/migration/versions/081_set_datastore_active.py

```diff
@@ -0,0 +1,54 @@
+import json
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text
+from pylons import config
+
+
+def upgrade(migrate_engine):
+
+    datastore_connection_url = config.get(
+        'ckan.datastore.read_url', config.get('ckan.datastore.write_url'))
+
+    if not datastore_connection_url:
+        return
+
+    datastore_engine = create_engine(datastore_connection_url)
+
+    try:
+
+        resources_in_datastore = datastore_engine.execute('''
+            SELECT table_name
+            FROM information_schema.tables
+            WHERE table_schema = 'public'
+            AND table_name != '_table_metadata'
```

Comment:
Should we get the list of tables from _table_metadata in case someone was clever and added their own tables to this db?